pom.xml

```diff
@@ -214,7 +214,6 @@
             <groupId>commons-lang</groupId>
             <artifactId>commons-lang</artifactId>
             <version>2.6</version>
-            <scope>test</scope>
```

Comment:
@prondzyn Let's keep the test scope (see my comment below on how to avoid needing this library).